src/utils/hooks.ts

```diff
@@ -0,0 +1,41 @@
+import { AppRootProps, NavModelItem } from '@grafana/data';
+import { pages } from 'pages';
```

Comment:
**Suggestion:** I would rather pass this in as a separate parameter to `useNavModel`.

I think the utility functions should be only depending on _more generic_ code, e.g. libs or constants, but not on more specifics ones like components or pages. This also helps to easily reuse this hook in projects with a different directory structure.

Reply:
Good point, done!